src/api/uploads/MultiputUpload.js

```diff
@@ -558,6 +557,11 @@ class MultiputUpload extends BaseMultiput {
                 fileId: this.fileId,
             };
             this.upload(uploadOptions);
+        } else {
+            // Handle internet disconnects (error.request && !error.response) and (!error.request)
+            // Also handle any 500 error messages
+            this.retryTimeout = setTimeout(this.getSessionInfo, 2 ** this.numResumeRetries * MS_IN_S);
```

Comment:
Should this be handled by the `retryNumOfTimes` util in the `utils/function.js` file?

Reply:
I think the code could be refactored to use it, but since it's already been tested and works, I think it may be better to keep it.  It was also existing code in box-ui-elements before my changes.